EverestSplash/EverestSplash.cs

```diff
@@ -330,43 +329,34 @@ private void LoadTextures() {
     private void HandleWindow() {
         SDL.SDL_ShowWindow(windowInfo.window);
 
-        // Animation values, SDL timers are a pain to use, this is easier
+        // Global loop vars
         int startEverestSpriteIdx = 0;
         string[] startingCelesteText = { // DO Make sure that the longest string goes first, for caching reasons
             "Starting Celeste...", "Starting Celeste", "Starting Celeste.", "Starting Celeste.."
         };
-        AnimTimer(500, () => {
-            windowInfo.startingEverestFontCache.SetText(startingCelesteText[startEverestSpriteIdx]);
-            startEverestSpriteIdx = (startEverestSpriteIdx + 1) % startingCelesteText.Length;
-        });
+        double textAccDt = 0;
```

Comment:
The variable name is a bit ambiguous and doesn't immediately convey its purpose. Maybe `textUpdateTimer`?